integration/airflow/tests/integration/docker/up.sh

```diff
@@ -56,4 +60,5 @@ pytest==6.2.2
 EOL
 
 docker-compose down
-docker-compose up -V --build --force-recreate --exit-code-from integration
+docker-compose up -V --build --abort-on-container-exit airflow_init postgres
+docker-compose up --build --exit-code-from integration --scale airflow_init=0
```

Comment:
I would chain these methods with `&&`, but also wondering why we need both build commands? Would be great to add some comments here ;)